app/controllers/api/framework_assessments_controller.rb

```diff
@@ -53,8 +53,10 @@ def supported_relationships
       assessment_serializer::SUPPORTED_RELATIONSHIPS
     end
 
-    def assessment
-      @assessment ||= assessment_class.find(params[:id])
+    def assessment(id = params[:id])
+      @assessment ||= assessment_class
+        .includes(active_record_relationships)
+        .find(id)
```

Comment:
These changes are helpful for the `show` method but probably not for `create` I think - so perhaps you don't need to pass in an optional `id` argument and this could be simplified further to

```suggestion
def assessment
    @assessment ||= assessment_class.includes(active_record_relationships).find(params[:id])
end
```

Reply:
please see above comment

Reply:
Discussed in person - keeping above `create` changes, so these can stay as is too.